example_project/run_simple_workflow.py

```diff
@@ -0,0 +1,235 @@
+    while ack not in ['y', 'n']:
+        ack = input('Please respond with "y" or "n": ').lower()
+    if ack == 'n':
+        print('Ok, maybe some other time!')
+        sys.exit()
+    print('Great!')
+
+
+def describe_workflow():
+    print('')
+    print('The Simple Workflow has two steps, one automated and one human.')
+    pause(2)
+    print('The automated step takes a URL and extracts a random image from '
+          'the page.')
+    pause(2)
+    print('The human step asks an expert to rate how "awesome" the image is '
+          'on a scale from one to five.')
+    pause(2)
+    print('For this demo, we will scrape an image from www.josephbotros.com, '
+          'the homepage of one of our rockstar developers.')
+    pause(2)
+    print('If you open that URL in a browser, it should be pretty obvious '
+          'which image the task will scrape.')
+    pause(2)
+
+
+def create_project():
+    print('')
+    print("Let's start by creating a new project that uses the workflow.")
+    pause(2)
+    print("We'll use the 'create_orchestra_project' API call.")
+    pause(2)
+    print('The call looks like this:')
+    print('''
+create_orchestra_project(
+  'simple_workflow',                       # The slug representing our workflow
+  'A test run of our simple workflow',     # A description of the new project
+  10,                                      # A priority level for the project
+  {
+    'url': 'http://www.josephbotros.com/'  # Data required by the workflow:
+  },
+)
+    ''')
+    pause(4)
+    print('Make sure you have the example project running in another window '
+          '(`python manage.py runserver`).')
+    input('Press enter when you are ready to make the call. ')
+
+    project_id = create_orchestra_project(
+        None,
+        'simple_workflow',
+        'A test run of our simple workflow',
+        10,
+        {
+            'url': 'http://www.josephbotros.com/'
+        },
+        'https://docs.google.com/document/d/1s0IJycNAwHtZfsUwyo6lCJ7kI9pTOZddcaiRDdZUSAs',  # noqa
+        'train',
+    )
+    print('Project with id {} created!'.format(project_id))
+    pause(2)
+    return project_id
+
+
+def project_info_1(project_id):
+    print('')
+    print('When we created our project, it immediately ran the first step of '
+          'the workflow: scraping an image from the website we passed in.')
+    pause(2)
+    print("Let's verify that this worked successfully by using Orchestra's "
+          "API to check the project info.")
+    pause(2)
+    print('The call looks like this:')
+    print('')
+    print('get_project_information(project_id)')
+    print('')
+    pause(2)
+    input('Press enter when you are ready to make the call. ')
+
+    project_info = get_project_information(project_id)
+    print("Information received! Here's what we got:")
+    print('')
+    pause(2)
+    pprint(project_info)
+    pause(10)
+    print('')
+    print("Note that 'tasks.crawl.status' is 'Complete', and "
```

Comment:
explain also that rate is not done yet, and is awaiting processing, which only happened after the crawl finished, and that we now have to have a human complete this step.